tests/slow/src/test/scala/tests/feature/Scala3CodeActionLspSuite.scala

```diff
@@ -1,27 +1,15 @@
 package tests.feature
 
 import scala.meta.internal.metals.BuildInfo
-import scala.meta.internal.metals.codeactions.ConvertToNamedArguments
-import scala.meta.internal.metals.codeactions.CreateCompanionObjectCodeAction
-import scala.meta.internal.metals.codeactions.ExtractMethodCodeAction
-import scala.meta.internal.metals.codeactions.ExtractRenameMember
-import scala.meta.internal.metals.codeactions.ExtractValueCodeAction
-import scala.meta.internal.metals.codeactions.FlatMapToForComprehensionCodeAction
-import scala.meta.internal.metals.codeactions.ImplementAbstractMembers
-import scala.meta.internal.metals.codeactions.InsertInferredType
-import scala.meta.internal.metals.codeactions.RewriteBracesParensCodeAction
-import scala.meta.internal.metals.codeactions.SourceOrganizeImports
-import scala.meta.internal.mtags.MtagsEnrichments.XtensionAbsolutePath
+import scala.meta.internal.metals.codeactions.InlineValueCodeAction
 
-import munit.Location
-import munit.TestOptions
 import tests.codeactions.BaseCodeActionLspSuite
 
 class Scala3CodeActionLspSuite
     extends BaseCodeActionLspSuite("cross-code-actions") {
 
   override protected val scalaVersion: String = BuildInfo.scala3
-
+  /*
```

Comment:
Could you uncomment the code? Or are these the tests you were having issues with?